### PR TITLE
Fix/filter playtime range

### LINKED ISSUE #72: As a gamer, I want to be able to filter playtime on a low and high, so that I can find a game within my time expectations easier
- [x] Filter with a span on playtime

### PR DESCRIPTION
**Pull request by x183**
Filters games on a range of playtimes instead of just a single number, fixes #72

### CODE REVIEW

**Comment by x183** [comment]
> Seems like branch was not based on main? And I will see with our stakeholder what the preferred way to filter amount of players is

Oopsie woopsie, something seems to have went wong, is fix now :+1:

**Review by Supergamer1337 — CHANGES_REQUESTED** [review]

File: backend/src/routers/gameRouter.ts [hunk 1]
```diff
@@ -208,8 +211,13 @@ gameRouter.post('/filter', validateRequestBody(filterGamesSchema), async (req, r
 	}
 	if (body.platform)
 		filter.platform = { name: body.platform };
-	if (body.playtime)
-		filter.playtimeMinutes = body.playtime;
+	if (body.playtimeMax || body.playtimeMin) {
+		filter.playtimeMinutes = {};
+		if (body.playtimeMax && filter.playtimeMinutes)
```
[the remark this answers is not in the document]
Comment (x183) [reply backend/src/routers/gameRouter.ts:216]: I totally agree with you, sadly Typescript does not :pensive:

[the remark this answers is not in the document]
Comment (Supergamer1337) [reply backend/src/routers/gameRouter.ts:216]: Worked fine for me when I removed it

**Review by Supergamer1337 — APPROVED** [review]
LGTM

**Comment by Krysset** [comment]
> Seems like branch was not based on main? And I will see with our stakeholder what the preferred way to filter amount of players is

Mistook PR for another.... Whoopsie

**Review by Krysset — APPROVED** [review]
Looks good!

**Pull request merged**